### PR TITLE
Json test improvements

### PR DESCRIPTION
**Pull request by shabtaisharon**

### CODE REVIEW

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 1]
```diff
@@ -182,10 +184,10 @@ public void getCompletedJob() throws Exception {
 
             final String expectedBeginning = "JobId: " + readJob.getJobId() + " | Status: COMPLETED | Bucket: " + bucketName
                     + " | Type: GET | Priority: HIGH | User Name: spectra | Creation Date: ";
-            final String expectedEnding = " | Total Size: " + objSize + " | Total Transferred: " + objSize;
+            final String expectedEnding = " | Total Size: " + objSize + " | Total Transferred: " ;//TODO add objSize when testing using BP 1.2 is not relevant anymore
 
             assertTrue(getJobResponse.getMessage().startsWith(expectedBeginning));
```
Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: Do we still need the startsWith and contains checks if we can check the items directly with the json models?

Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: this is a different test

Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: Oh I see, my mistake

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/helpers/JobResponse.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        return status;
+    }
+
+    public Data getData() {
+        return data;
+    }
+
+    public static class Meta {
+        @JsonProperty("Date")
+        private String date;
```
Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/helpers/JobResponse.java]: This doesn't have a get method.

Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/helpers/JobResponse.java]: I removed all the get & set functions we don't use

Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/helpers/JobResponse.java]: I would at least leave the gets in place

Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/helpers/JobResponse.java]: don't you think we should add them if we need them?

**Pull request merged**